detekt.yml

```diff
@@ -84,7 +84,7 @@ formatting:
     active: true
     autoCorrect: true
   ExpressionBodySyntaxLineBreaks:
-      active: true
+      active: false
```

Comment:
The train of thought is that autofix > MaxLineLength + regenerate baseline for consistency purposes. It's quite annoying for now, so reverting would mean fixing all the MaxLineLength back. No more than 30 minutes to do it tho.